src/Microsoft.DotNet.ProjectModel/CommonCompilerOptions.cs

```diff
@@ -161,6 +173,26 @@ public static CommonCompilerOptions Combine(params CommonCompilerOptions[] optio
                 {
                     result.OutputName = option.OutputName;
                 }
+
+                if (option.CompilerName != null)
+                {
+                    result.CompilerName = option.CompilerName;
+                }
+
+                if (option.CompileInclude != null)
+                {
+                    result.CompileInclude = option.CompileInclude;
+                }
+
+                if (option.EmbedInclude != null)
+                {
+                    result.EmbedInclude = option.EmbedInclude;
+                }
+
+                if (option.CopyToOutputInclude != null)
+                {
+                    result.CopyToOutputInclude = option.CopyToOutputInclude;
+                }
```

Comment:
these are complex types, i'm not sure about the combine logics here. you're sure it is just simply replace them?